voidcrew/modules/cloning/code/cloning_circuitboards.dm

```diff
@@ -0,0 +1,46 @@
+//Computer
+
+/obj/item/circuitboard/computer/prototype_cloning
+	name = "Prototype Cloning (Computer Board)"
+	icon_state = "medical"
+	build_path = /obj/machinery/computer/prototype_cloning
+
+/obj/item/circuitboard/computer/cloning
+	name = "Cloning (Computer Board)"
+	icon_state = "medical"
+	build_path = /obj/machinery/computer/cloning
+
+//Machinery
+
+/obj/item/circuitboard/machine/clonepod
+	name = "Clone Pod (Machine Board)"
+	icon_state = "medical"
```

Comment:
Circuitboards don't have an icon state anymore, they have a GAGS color, look at other medical machinery to see